### PR TITLE
Remove mistral from installation section of documents

### PR DESCRIPTION
**Pull request by amanda11**
Mistral has been removed from StackStorm as part of 3.3 roadmap, update the install chapter of the documentation to remove mistral and postgreSQL references.
Not included: puppet installation changes

Addresses part of the st2docs section of https://github.com/StackStorm/st2/issues/4762

### CODE REVIEW

File: docs/source/install/uninstall.rst [hunk 1]
```diff
@@ -40,64 +40,34 @@ below. Only execute the instructions for your distribution.
 1. Stop Services
 ----------------
 
-* Ubuntu 16.04:
+* Ubuntu 16.04/18.04:
```
Comment (armab) [inline docs/source/install/uninstall.rst:43]: :+1:

File: docs/source/upgrade_notes.rst [hunk 2]
```diff
@@ -17,6 +17,11 @@ Upgrade Notes
     trigger:
       type: "core.st2.action.file_written"
 
+* Support for Mistral workflows was removed. Before upgrading to v3.3, ensure all Mistral workflows have been converted to Orquesta workflows. A tool is available for assisting in this conversion, more information can be found in the ``orquestaconvert`` `README.md <https://github.com/StackStorm/orquestaconvert/blob/master/README.md>`_.
```
Comment (armab) [inline docs/source/upgrade_notes.rst:20]: :+1:

Comment (armab) [reply docs/source/upgrade_notes.rst:20]: @nmaludy I hope we'll get covered it more soon in the Orquesta Convert blog.

**Comment by winem** [comment]
Looks good to me!

**Comment by nmaludy** [comment]
@armab do you know if we still have the Visio drawings for the architectural diagrams?

I believe the following need updated:

- https://docs.stackstorm.com/install/overview.html
- https://docs.stackstorm.com/reference/ha.html
- https://docs.stackstorm.com/overview.html

**Comment by armab** [comment]
@nmaludy The st2mistral documentation work is split into several PRs. Check also https://github.com/StackStorm/st2docs/pull/1012 https://github.com/StackStorm/st2docs/pull/1011 that updates different documentation sections.

@nmaludy RE Diagrams, check the https://github.com/StackStorm/st2docs/pull/1010 which is part of updating that. I think @blag was helping @amanda11 @winem to get these diagrams re-built.

**Comment by amanda11** [comment]
I saw a comment that @blag had updated the HA deployment big picture and the deployment single box drawings. @winem did you get access to the updated pictures, as i think you originally had access denied issues (I also can't access the google links that had them in).

**Comment by armab** [comment]
Good to merge? @amanda11 feel free when you think it's ready.

**Pull request merged**